MyLibrary/Tests/MyLibraryTests/MockWeatherService.swift

```diff
@@ -10,6 +11,23 @@ class MockWeatherService: WeatherService {
         self.shouldReturnTemperatureWithAnEight = shouldReturnTemperatureWithAnEight
     }
 
+    func getWeather(completion: @escaping (_ response: Result<Int, Error>) -> Void) {
+        switch (shouldSucceed, shouldReturnTemperatureWithAnEight) {
+        case (true, true):
+            let temperatureInFarenheit = 38
+            completion(.success(temperatureInFarenheit))
+
+        case (true, false):
+            let temperatureInFarenheit = 39
+            completion(.success(temperatureInFarenheit))
+
+        case (false, _):
+            let error404 = AFError.explicitlyCancelled
+            completion(.failure(error404))
+        }
+    } 
+
+    /*
```

Comment:
Adding code just to comment it out can lead to messy code later on. Not a good practice.